content/en/core/releases/4.1.0.md

```diff
@@ -0,0 +1,70 @@
+---
+title: "4.1.0 release notes"
+linkTitle: "4.1.0"
+weight:
+description: >
+relevantLinks: >
+toc_hide: true
+---
+
+## Known issues
+
+Check the repository for the [latest known issues](https://github.com/medic/cht-core/issues?q=is%3Aissue+label%3A%22Affects%3A+4.1.0%22).
+
+## Upgrade notes
+
+### Breaking changes
+
+None.
+
+### UI/UX changes
+
+- [#7778](https://github.com/medic/cht-core/issues/7778): Bulk delete of reports redesign
+
+## Highlights
+
+### Support replacing a CHW user without any connectivity
+
+Offline users can now be replaced on a device so that a new user can use that device without needing to immediately sync with the server. You can read more about it in [the docs]({{< ref "apps/reference/app-settings/transitions#create_user_for_contacts" >}}).
+
+### Redesigned bulk delete of reports redesign
+
+The bulk delete of Reports tab has been redesigned to improve the user experience. Check out the new UI in the [Reports tab docs]({{< ref "apps/features/reports/#bulk-delete-reports" >}}).
+
```

Comment:
It's a bit weird to have this listed under UI/UX and Highlights. I think this section should be moved up into the UI/UX section otherwise I'm left wondering what actually changed!

Also thanks for providing the link to the docs. I think it'd be even better to also include the screenshots inline, otherwise release notes are really bland.

Reply:
Great suggestions, thanks! How is it now?